scripts/core/editor3/components/BaseUnstyledComponent.tsx

```diff
@@ -16,6 +16,17 @@ export function getEditorBlock(event) {
     return event.originalEvent.dataTransfer.getData(EDITOR_BLOCK_TYPE);
 }
 
+// Dragging a single line in chrome removes DraftJS's data
+// and only sends a <span> containing the dragged text next
+// to a <meta> tag
+function htmlIsPlainTextDragged(html: string): boolean {
+    const parser = new DOMParser().parseFromString(html, 'text/html');
+
+    return parser.querySelectorAll('span').length === 1 &&
+        parser.querySelectorAll('meta').length === 1 &&
+        parser.querySelectorAll(':not(span):not(meta):not(html):not(head):not(body)').length === 0;
```

Comment:
What if it contains a link or a div. Wouldn't it be better to check by locating the root span and comparing `innerHTML` with `innerText`?

Reply:
but a link or a div probably indicates that an `embed` should be created shouldn't it?

Reply:
Probably. But if you have `<span><a href="...">...</a><span>` your function will determine that it's plain text, won't it? That's why I suggest comparing `innerHTML` / `innerText` instead.

Reply:
It won't determine that's plain text because of the last check. I'll add a test to confirm that. We can keep adding cases in case we ever run into this issue again.

Reply:
I now see it won't. It would think that `<span><body>text</body></span>` is plain text though. I know it's invalid HTML, but why wouldn't you use `innerHTML`/`innerText` comparison?

The current condition looks like checking if a button is green, by adding a condition that it's not blue, not red, not yellow etc.

Reply:
it's more like checking it's green and it's not any other color than green, which sounds right to me.

What would you do with the innerHTML? If you create a DOMParser it will still add a html/body/head tag

Reply:
If I understand correctly, you want to check that HTML contains a single span which itself only contains plain text. This looks more clear and straight-forward to me:

```
var doc = new DOMParser().parseFromString('...', 'text/html');
doc.body.childElementCount === 1 && doc.body.childNodes[0].tagName === 'SPAN' && doc.body.childNodes[0].innerHTML === doc.body.childNodes[0].innerText
```